recipes/gnuplot/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./configure \
+    --prefix=$PREFIX \
+    --without-x \
+    --without-lua
```

Comment:
We don't have `wx`, though I believe `wxpython` does exist on `defaults`. Not sure if we need the `--without-wx` flag or not.